terraform/deployments/govuk-publishing-infrastructure/asset_manager_s3.tf

```diff
@@ -6,17 +6,21 @@ resource "aws_iam_policy" "asset_manager_s3" {
     Version = "2012-10-17"
     Statement = [
       {
-        Effect   = "Allow"
-        Action   = "s3:ListBucket"
+        Effect = "Allow"
+        Action = [
+          "s3:GetBucketLocation",
+          "s3:ListBucket",
+        ]
         Resource = data.terraform_remote_state.infra_assets.outputs.asset_manager_bucket_arn
       },
       {
         Effect = "Allow"
         Action = [
-          "s3:PutObjectAcl",
-          "s3:PutObject",
-          "s3:GetObject",
-          "s3:DeleteObject",
```

Comment:
Delete is not needed?

Reply:
Seems to be, e.g. [here](https://github.com/alphagov/asset-manager/blob/f4d0b87/lib/tasks/assets.rake#L2) and [here](https://github.com/alphagov/asset-manager/blob/f4d0b874/lib/s3_storage.rb#L37). There's a [DELETE method in the API](https://github.com/alphagov/asset-manager/blob/f4d0b874/lib/s3_storage.rb#L37) for the `assets/:id` resource.

Reply:
Oh, just realised I might have misinterpreted your question. Delete is covered by the wildcard in `s3:*Object`.